assets/dev/js/editor/elements/views/base.js

```diff
@@ -16,7 +16,7 @@ BaseElementView = BaseContainer.extend( {
 
```

Comment:
Maybe you should take the opportunity and convert the whole class to a complete ES6's style

Reply:
Waiting for "Empty trays" sprint